resistbot-gateway-forwarder/src/index.ts

```diff
@@ -14,7 +15,7 @@ const client = new Client({
 client.on(Events.MessageCreate, async (message) => {
     console.log('Got message', message);
     console.log('Got message JSON', message.toJSON());
-    fetch('http://flask-backend:5000/incoming-dm', {
+    fetch(`${BACKEND_URL}:5555/incoming-dm`, {
```

Comment:
I'd say make this port also an ENV is possible?

I'm also thinking we want these paths defined someplace central like a conf file. So that in unit tests etc we can reference them. Couple more below but I wont' duplicate my comments..

Reply:
Moved the port to an ENV. I skipped moving the paths to a conf file for now. I figure we can do that when we add tests.